Assignment2/printArray.js

```diff
@@ -0,0 +1,4 @@
+function printArray(array){

```

Comment:
argument name shouldn;t be "array". It should be adjective like "userList" or. "productList" or "userArray"

Reply:
Actually we were given this exercise on codewars platform there they mentioned it as array and they asked to code below.
I will change this.

Reply:
ok